CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [0.2.0] 2019-07-29
+
+### Added
+ - Supported version of Terraform is 0.12. [#10]
```

Comment:
why link is "10" but PR is "9"? :) it's not important of course

Reply:
Thank you, Andrii! [Fixed](https://github.com/terraform-google-modules/terraform-google-vpc-service-controls/commit/681718b5474a860493c68f5762f49c4113b52efa)